src/cross_auth/_session.py

```
@@ -0,0 +1,129 @@
+        valid = False
+
+    if not valid:
+        return None
+
+    return user
+
+
+def create_session(
+    user_id: str,
+    storage: SecondaryStorage,
+    max_age: int = 86400,
```

Ruff (0.14.14)

[warning] Unused function argument: `max_age`

(ARG001)